onnxruntime/core/graph/model.cc

```diff
@@ -139,11 +139,18 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
 
   auto domain_map = schema_registry->GetLatestOpsetVersions(false);
   for (const auto& domain : domain_map) {
-    if (domain_to_version.find(domain.first) == domain_to_version.end()) {
+    auto iter = domain_to_version.find(domain.first);
```

Comment:
If we plan to consume the latest ONNX commits before the release (and thus taking in partial opset-12 schemas) to develop opset-12 ops and unblock converter testing then the domain to version map from ONNX would return 12 for the ONNX domain. I guess we need to find a way to re-usable mechanism to clamp it to a lower version that ORT claims full support for in a release ? (In this case 11)

Reply:
It can be solved in the GetLatestOpsetVersions function.

Reply:
Sure.

Reply:
Yeah, this PR will disallow opset 12 models from loaded until we update the onnx commit. Once we update onnx, this won't work unless we maintain the opset version we support in a given release somewhere and check against that. While we're in the master branch, we can store opset 12 in this MAX_SUPPORTED_OPSET variable to allow testing of opset 12 op implementations by others. When we cut a release, we should use opset 11 for this variable. Would this work? Is this PR going to address this?